internal/backend/workloadproxy/reconciler.go

```diff
@@ -25,8 +26,27 @@ import (
 )
 
 type lbStatus struct {
-	lb                *loadbalancer.TCP
-	upstreamAddresses []string
+	lb *loadbalancer.TCP
+}
+
+func (lbSts *lbStatus) Close() error {
+	var lb *loadbalancer.TCP
```

Comment:
do we need a nil check for `lbStatus`>?

Reply:
I think it can't be nil (as we check before we call this func)